### DISCUSSION TITLE
How do I upload my website to PocketHost to be served from the pb_public folder?

### DISCUSSION THREAD

**Discussion by aabmets** [pockethost/pockethost#150] [Q&A] [answered]
The question is in the title.

**Comment by benallfree** [comment, marked as answer]
@aabmets `pb_public` is not supported at this time, it will probably come around the same time as #25.

Can you tell me more about your use case? Are you wanting to host your frontend at `https://fubar.pockethost.io` or something like that?

**Reply by aabmets** [reply]
I'm auditioning for a job position and the potential employer gave me a test work which requires the usage of PocketBase. I don't have a problem with using PocketBase locally, I just thought I could give the employer a nice link to interact with my test work without them having to run the PocketBase server locally. This would just be sugar on top of the cake, as they asked to run the PocketBase server themselves anyways.

**Reply by benallfree** [reply]
An employer is asking for PocketBase experience? @ganigeorgiev might like to know about that story :)

**Reply by aabmets** [reply]
Not really, the focus is on the candidates ability to create an authenticated chat app. PocketBase is used to evaluate the candidates ability to familiarize themselves with unknown systems, because PocketBase is obscure and unknown enough so that the majority of developers haven't heard anything about it.
